ts-rs/tests/recursion_limit.rs

```diff
@@ -1,170 +1,82 @@
+use std::any::TypeId;
+
+use ts_rs::{
+    typelist::{TypeList, TypeVisitor},
+    TS,
+};
+
+#[rustfmt::skip]
 #[derive(Debug, ts_rs::TS)]
-#[ts(export)]
+#[ts(export, export_to = "very_big_types")]
```

Comment:
Missing trailing slash here

Reply:
:D you were right that this is a big footgun

Reply:
Sidenote, I didn't realize this would break the format feature. I guess this confirms they use the path only to check the extension for `.ts`